src/applications/claims-status/tests/components/ClosedClaimMessage.unit.spec.jsx

```diff
@@ -1,63 +1,128 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
+import { render } from '@testing-library/react';
 import moment from 'moment';
 
 import ClosedClaimMessage from '../../components/ClosedClaimMessage';
 
 describe('<ClosedClaimMessage>', () => {
-  it('should render closed claims within 30 days', () => {
-    const claims = [
-      {
-        id: 1,
-        type: 'evss_claims',
-        attributes: {
-          dateFiled: '2023-01-01',
-          open: false,
-          phaseChangeDate: moment()
-            .add(-29, 'days')
-            .format('YYYY-MM-DD'),
+  context('EVSS claims', () => {
+    it('should render closed claims within 30 days', () => {
+      const claims = [
+        {
+          id: 1,
+          type: 'evss_claims',
+          attributes: {
+            dateFiled: '2023-01-01',
+            open: false,
+            phaseChangeDate: moment()
+              .add(-29, 'days')
+              .format('YYYY-MM-DD'),
+          },
         },
-      },
-    ];
-    const tree = SkinDeep.shallowRender(<ClosedClaimMessage claims={claims} />);
+      ];
+      const tree = SkinDeep.shallowRender(
```

Comment:
Imported in `render` but this is still using SkinDeep?

Reply:
I didn't want to mess with the existing tests, but since I wrapped them in a context they show as being code changes. And yeah I used a mix or testing library and enzyme for some of these